### PR TITLE
[ADD] add crm_lead_address_street3 addon

### PR DESCRIPTION
**Pull request by rdeheele**
It adds a street3 field on lead addresses.
It depends on partner_address_street3 addon.

### CODE REVIEW

File: crm_lead_address_street3/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Street3 in CRM Lead Addresses
+=============================
+
+Add a third field to the lead addresses.
+Check that this field is well passed when the
+lead is converted in an opportunity.
+
```
Comment (pedrobaeza) [inline crm_lead_address_street3/README.md]: Mentions in Installation section that this need the module partner_address_street3, that it's on OCA/partner-contact...

Comment (rdeheele) [inline crm_lead_address_street3/README.md]: @pedrobaeza thanks and i'll add too a git clone in .travis.yml install part

File: crm_lead_address_street3/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+{
+    'name': 'Street3 in lead addresses',
+    'version': '0.1',
+    'author': 'Camptocamp',
+    'maintainer': 'Camptocamp',
+    'category': 'Customer Relationship Management',
+    'complexity': 'easy',
+    'depends': ['partner_address_street3', 'crm'],
+    'description': """Add a third field to the lead addresses
+                      Check that this field is well passed when the
```
Comment (nbessi) [inline crm_lead_address_street3/__openerp__.py]: please remove the description key as there is a readme

File: crm_lead_address_street3/model/crm_lead.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+##############################################################################
+from openerp import models, fields
+
+
+class crm_lead(models.Model):
+    """Add third field in lead address"""
+
+    _inherit = "crm.lead"
+
+    def _lead_create_contact(self, cr, uid, lead, name, is_company,
```
Comment (nbessi) [inline crm_lead_address_street3/model/crm_lead.py:30]: I would add an api.v7

File: crm_lead_address_street3/model/crm_lead.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                             parent_id=False, context=None):
+        partner_obj = self.pool['res.partner']
+        partner = super(crm_lead, self).\
+            _lead_create_contact(cr, uid, lead, name, is_company,
```
Comment (nbessi) [inline crm_lead_address_street3/model/crm_lead.py]: weird indent '\' is not needed

File: crm_lead_address_street3/test/test_street3.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  In order to test that the street3 is well passed to the future partner
+  I create a new lead
+-
+  !record {model: crm.lead, id: lead_test}:
+    name: Jester Smurf
+    contact_name: Jester Smurf
+    street: Sarsaparilla Street
+    street2: Cursed Country
+    street3: Third Mushroom
+    city: Schtroumpf les Bains
```
Comment (nbessi) [inline crm_lead_address_street3/test/test_street3.yml:11]: You know the place too :tongue:

**Comment by nbessi** [comment]
Some details to fix but globally :+1:

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**